docs/TROUBLE.md

```diff
@@ -0,0 +1,173 @@
+# mLRS Documentation: Diagnosing Common Problems #
+
+([back to main page](../README.md))
+
+This page attempts to provide a step by step guide to isolate and correct some common problems.  It covers only a core use case where an OpenTX or EdgeTX radio with a mLRS Tx module is used to communicate with a mLRS receiver connected to an ArduPilot flight controller to provide both RC control and MAVLink Telemetry to a ground station and/or [Yaapu Telemetry App](CRSF.md).  Users with other use cases in mind may still find some parts of this page helpful.
+
+Numbered topics here assume correct verification of previous functionality so don't skip ahead.  You must establish correct operation for each numbered topic in turn or the deductions we make later may not be correct!  Each numbered topic also assumes the state and configuration established in previous topics.  For example, topic 3 instructs you to power on both your receiver and Tx module and verify the connection.  All subsequent topics assume they are powered on and connected.
+
+Each numbered topic is structured as a test procedure with expected results followed by a list of things to check or tests to perform to help locate the problem if the expected test results are not observed.
+
+Some notation and terminology:
```

Comment:
I think this could be compactified somewhat.
For instance:

Some notation and terminology:
- We use "Tx module" and "receiver" to refer to a mLRS module in the Tx/transmitter role and Rx/receiver role.  In the use case assumed here, the Tx module will be installed in a OpenTX or EdgeTX radio handset, and the receiver will be mounted on the vehicle and connected to the flight controller.
- We use lower case "rx" and "tx" when refering to the receive abd transmit pads or pins of a serial port connection.

Reply:
(note also no blank lines between the bullets)

Reply:
Should I remove blank lines between bullets in the rest of the document as well?

Reply:
It looks fine to me without the extra space between bullet items so I think I'll remove them for all bullet lists.